docs/platforms/python/configuration/options.mdx

```diff
@@ -111,7 +111,7 @@ If you enable this option, be sure to manually remove what you don't want to sen
 
 <ConfigKey name="event-scrubber">
 
-If <PlatformIdentifier name="send-default-pii" /> is turned off, scrubs the event payload for sensitive information from a `denylist`. See how to [configure the scrubber here](../../data-management/sensitive-data/#event-scrubber).
+Scrubs the event payload for sensitive information such as cookies, sessions and passwords from a `denylist`. It can additionally be used to scrub from another `pii_denylist` if <PlatformIdentifier name="send-default-pii" is disabled. See how to [configure the scrubber here](../../data-management/sensitive-data/#event-scrubber).
```

Comment:
```suggestion
Scrubs the event payload for sensitive information such as cookies, sessions, and passwords from a `denylist`. It can additionally be used to scrub from another `pii_denylist` if <PlatformIdentifier name="send-default-pii" is disabled. See how to [configure the scrubber here](../../data-management/sensitive-data/#event-scrubber).
```